### DISCUSSION TITLE
State field does not work

### DISCUSSION THREAD

**Discussion by pippo73** [photoprism/photoprism#3666] [Q&A]
Hi all, 
I don't know if it is a bug or not, so I start to write here.
I've noticed a strange behaviour of the system.
I've selected several photos. Editing the datas, I've added the state where they have been taken.
I was expecting that in the page places=>states I will find the state, and therefore the photos I've indicated.
This does not happend :(
Could anyone confirm this behavior also on your servers? Do you think it is correct?

thank all

Leo

**Comment by lastzero** [comment]
The names of states and cities are set automatically based on the GPS coordinates. Did you change the location or how did you set a state?

**Reply by pippo73** [reply]
No, the images are without gps coordinates.
I've just choosed a state in the drop-down list

**Reply by lastzero** [reply]
The dropdown is for countries, not states? You can use the Country filter in the regular search to only find pictures from a specific country.

**Reply by pippo73** [reply]
I do an example hope to be more clear.
I take a picture without gps coordinates
![Screenshot_20230824-132236_Chrome](https://github.com/photoprism/photoprism/assets/757121/563f6160-2f7e-403b-a43b-3220638ad1ca)
I go in the infos of the immage and I select a state from the drop-down list.
Once done, I expect that this state appears in the list
![Screenshot_20230824-132048_Chrome](https://github.com/photoprism/photoprism/assets/757121/9db4d0d2-28ab-4aba-a093-f6068191d188)
And to find all the pics of this state to be piled togheter
![Screenshot_20230824-132115_Chrome](https://github.com/photoprism/photoprism/assets/757121/0db1de61-4571-440b-af03-6a2e076257bd)

Sorry for the bad quality of the pics, but I took them from the phone to reply fast. Hope they give you the idea.

**Reply by pippo73** [reply]
I don't expect to see them on the map (or maybe you can set in the middle of the country or on top of a generic position on the capital of the country) but I think it should be in the list of the states.

**Reply by lastzero** [reply]
In Japan, a state would be the same as a prefecture. That information is missing without coordinates, so it cannot be displayed in this specific view.

**Reply by pippo73** [reply]
That's the problem. Only USA, CH, and maybe a few other have difference between state and country. I think it makes more sense to act as I say. I will try to submit a change request...

**Reply by lastzero** [reply]
We can rename state to region so that it is not confused with the surrounding country? We already have a country filter and country dropdown in the search form. It only includes countries for which there are images. So the ability to see what countries there are already exists. Also, you can view the images that were taken in a specific country.

**Reply by pippo73** [reply]
I've check out now with the pc. Part of my "problem" is related to the translation from english to italian. In english makes more sence.
the right translation is "states" = > "regioni" not "stati" (which means countries)

But apart to the translation problems, I think it has more sense to identify the countries instead of the states.

**Reply by lastzero** [reply]
This view is specifically for countries where states are important, such as the United States. We may later add a similar view that shows countries, similar to the Moments which cluster country and year.

**Reply by pippo73** [reply]
Thank you for your fast and efficient support!
I've seen that you have the country filter in the "general search" page.
Then I invite you to add such filter also in the "states page" where you can choose also the country as a filter :-)
